scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,190 @@
+@import 'settings';
+
+@mixin divider-position($divider-order, $large-screen: false) {
+  @if $large-screen {
+    @if $divider-order == 1 {
+      &.p-equal-height-row--has-1st-divider::before {
+        @content;
+      }
+    }
+
+    @if $divider-order == 2 {
+      &.p-equal-height-row--has-2nd-divider::after {
+        @content;
+      }
+    }
+
+    @if $divider-order == 3 {
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider)::before,
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider)::after {
+        @content;
+      }
+    }
+  } @else {
+    @if $divider-order == 1 {
+      &.p-equal-height-row--has-1st-divider .p-equal-height-row__col::before {
+        @content;
+      }
+    }
+
+    @if $divider-order == 2 {
+      &.p-equal-height-row--has-2nd-divider .p-equal-height-row__col::after {
+        @content;
+      }
+    }
+
+    @if $divider-order == 3 {
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider) .p-equal-height-row__col::before,
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider) .p-equal-height-row__col::after {
+        @content;
+      }
+    }
+  }
+}
+
+@mixin divider-styles($large-screen: false) {
+  // For each row or column grid we only have access to two pseudo elements
+  // if 1st-divider (::before) is present, assume 2nd-divider (::after) isn't, then 3rd-divider must be (::after)
+  // if 2nd-divider (::after) is present, assume 1st-divider (::before) isn't, then 3rd-divider must be (::before)
+  @if $large-screen {
+    &.p-equal-height-row--has-1st-divider::before,
+    &.p-equal-height-row--has-2nd-divider::after,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider)::before,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider)::after {
+      @content;
+    }
+  } @else {
+    // For smaller screens, the divider pseudo elements are inserted at the column level
+    &.p-equal-height-row--has-1st-divider .p-equal-height-row__col::before,
+    &.p-equal-height-row--has-2nd-divider .p-equal-height-row__col::after,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider) .p-equal-height-row__col::before,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider) .p-equal-height-row__col::after {
+      @content;
+    }
+  }
+}
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
```

Comment:
Does this need to be a child of a row? Because I guess that's the only way for the subgrid to work?

I think we discussed this as independent component, with any possible parent (for example we need to be able to wrap it in `p-section`). So the root element of the component can't use subgrid I'm afraid. It needs to extend the `%row`, so that it defines it's own columns (that are aligned with the rest of the grid).

Reply:
no it doesn't have to be and extending %vf-row does remove the dependency of having to wrap this component inside of `.row`. I can update this and relevant html + docs sections, although to use it inside a 25-75 grid layout remains the same since it's no different than having a `.row` nested.